client/sagas/err-saga.js

```diff
@@ -1,20 +1,14 @@
-//   () =>
-//     (store: Store) =>
-//     (next: (action: Action) => Object) =>
-//     errSaga(action: Action) => Object|Void
-export default () => ({ dispatch }) => next => {
-  return function errorSaga(action) {
-    const result = next(action);
-    if (!action.error) { return result; }
-
-    console.error(action.error);
-    return dispatch({
+export default function errorSaga(action$) {
+  return action$
+    .filter(({ error }) => !!error)
+    .map(({ error }) => error)
+    .doOnNext(error => console.error(error))
```

Comment:
In the doOnNext method call you are passing in a function. If you pass in the method from the console object it loses it's internal context (the `this` object internally becomes the `this` object of the callee function). 

Binding it to the console object ensures that its internal context cannot change.